packages/destination-actions/src/destinations/intercom/groupIdentifyContact/index.ts

```diff
@@ -0,0 +1,151 @@
+import { ActionDefinition, ModifiedResponse, RequestClient } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import { getUniqueIntercomContact } from '../util'
+import type { Payload } from './generated-types'
+
+interface IntercomCreateCompanyData {
+  id: string
+}
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Identify Company',
+  description: 'Create or update a company in Intercom and attach a contact.',
+  defaultSubscription: 'type = "group"',
+  fields: {
+    remote_created_at: {
+      type: 'datetime',
+      description: 'The time the company was created by you.',
+      label: 'Company Creation Time',
+      default: {
+        '@path': '$.traits.createdAt'
+      }
+    },
+    external_id: {
+      type: 'string',
+      description:
+        'A unique identifier for the contact generated outside Intercom. External ID is required if the role is `user` and email is blank.',
+      label: 'External ID',
+      default: {
+        '@path': '$.userId'
+      }
+    },
+    email: {
+      type: 'string',
+      description: "The contact's email address. Email is required if the role is `user` and external ID is blank.",
+      label: 'Email Address',
+      format: 'email',
+      default: {
+        '@path': '$.traits.email'
+      }
+    },
+    company_id: {
+      type: 'string',
+      description: "The unique identifier of the company. Once set, this can't be updated.",
+      label: 'Company ID',
+      required: true,
+      default: {
+        '@path': '$.groupId'
+      }
+    },
+    contact_id: {
+      type: 'string',
+      description:
+        'The unique identifier for the contact which is given by Intercom. Setting a Contact ID will attach this contact to the company.',
```

Comment:
@NolanChan Since we also use email and/or external ID to help attach a contact to a company I might suggest an update to the description. Thoughts?

```suggestion
        'The unique identifier for the contact which is given by Intercom. Contact ID can be used to attach a contact to a company instead of external ID or email.',
```